tensorboard/webapp/metrics/views/card_renderer/image_card_component.ng.html

```diff
@@ -21,6 +21,9 @@
       title="{{ tag }}"
       value="{{ title }}"
     ></tb-truncated-path>
+    <vis-selected-time-warning
+      *ngIf="selectedTime && selectedTime.clipped"
```

Comment:
Would it be worth setting "isClipped" property?

https://cs.opensource.google/tensorflow/tensorboard/+/master:tensorboard/webapp/metrics/views/card_renderer/vis_selected_time_warning_component.ts;l=40;drc=ea35c6034141ae4c0fbdd23716539a57d5447dc2

Your TODO makes it seem like you don't want the default to be true.